formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/serializers/ConfigMemorySizeSerializer.kt

```diff
@@ -0,0 +1,70 @@
+package kotlinx.serialization.hocon.serializers
+
+import com.typesafe.config.*
+import java.math.BigInteger
+import kotlinx.serialization.*
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.encoding.*
+import kotlinx.serialization.hocon.*
+
+/**
+ * Serializer for [ConfigMemorySize].
+ * For decode using [HOCON Size in bytes format](https://github.com/lightbend/config/blob/main/HOCON.md#size-in-bytes-format).
```

Comment:
I'm sorry, I can't quite understand what you mean here. Are you saying decoding accepts all possible values from the spec (including powers of two and ten)? And encoding always emits powers of two?

Reply:
Yes, when decoding, can use all formats from the specification. For encoding, I selected on one option - to use powers of two, as it is easier to implement.

Reply:
Bump: Rewrite as 'All possible Hocon size formats....'

Reply:
Fixed